com.unity.template-universal/Packages/com.unity.template.universal/package.json

```diff
@@ -7,6 +7,14 @@
   "host": "hub",
   "description": "The Universal Render Pipeline Template configures Project settings for Projects where performance, wide platform support, and ease of customizing graphics are the primary considerations.\n\nThis Template uses the Universal Render Pipeline (URP) and Shader Graph. URP is prebuilt Scriptable Render Pipeline that is quick and easy to customize, and lets you create optimized graphics across a wide range of platforms. URP also includes an optimized 2D renderer complete with 2D lights and pixel perfect rendering, and an integrated post-processing solution. Shader Graph is a tool that allows you to create shaders using a visual node editor instead of writing code.\n\nThis template contains a sample Scene that contains examples of how to configure lighting settings, Materials, Shaders, and post-processing effects in URP, several preconfigured Universal Render Pipeline Assets that let you quickly swap between graphics quality levels, and Presets that have been optimized for use with URP.",
   "dependencies": {
-    "com.unity.render-pipelines.universal": "13.0.0"
+    "com.unity.2d.sprite": "1.0.0",
+    "com.unity.2d.tilemap": "1.0.0",
+    "com.unity.collab-proxy": "1.7.1",
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.render-pipelines.universal": "13.0.0",
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.timeline": "1.6.2",
+    "com.unity.ugui": "1.0.0",
+    "com.unity.xr.legacyinputhelpers": "2.1.8"
```

Comment:
Same as my comment about the 2d packages above

Reply:
@phi-lira  / @Verasl

Reply:
This is will need someone more familiar with XR to chime in, looking a the name, it should be removed, but @sandy-carter-unity or @thomas-zeng will know for sure.

Reply:
If it should be removed it should be removed from manifest.json as well as otherwise that package will be imported.

Reply:
That package is still being used in URP XR. It's a very bad name since it has Legacy in it, but yeah it's something needed for setting up an XR Rig if you aren't using the XR Interaction Toolkit. The main behavior that package has is the TrackedPoseDriver.

Reply:
thanks for the info.